src/dbnode/integration/setup.go

```diff
@@ -1035,6 +1035,13 @@ func newClients(
 		verificationAdminOpts = verificationAdminOpts.SetEncodingProto(prototest.ProtoPools.EncodingOpt).(client.AdminOptions)
 	}
 
+	if len(opts.CustomAdminOptions()) > 0 {
+		for _, opt := range opts.CustomAdminOptions() {
```

Comment:
maybe optFn? Found myself scratching my head for a sec on this. Feel free to ignore if this is the standard

Reply:
It _is_ the standard but that doesn't mean we should keep doing it :). Will update.